client/state/selectors/get-past-billing-transaction.js

```diff
@@ -20,8 +20,17 @@ import getPastBillingTransactions from './get-past-billing-transactions';
  * @param  {String}  id      ID of the transaction
  * @return {?Object}         The transaction object
  */
-const getPastBillingTransaction = createSelector( ( state, id ) => {
-	return find( getPastBillingTransactions( state ), { id } ) || null;
-}, getPastBillingTransactions );
+const getPastBillingTransaction = createSelector(
+	( state, id ) => {
+		return (
+			find( getPastBillingTransactions( state ), { id } ) ||
+			get( state, [ 'billingTransactions', 'individualTransactions', 'receipts', id ], null )
+		);
+	},
+	state => [
+		getPastBillingTransactions( state ),
+		...keys( get( state, [ 'billingTransactions', 'individualTransactions', 'receipts' ] ) ),
```

Comment:
I think this will be calculated every time, regardless of whether the receipt have changed. Can we optimize it?